components/nexus-common/src/main/java/org/sonatype/nexus/common/text/Strings2.java

```diff
@@ -10,30 +10,22 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.security.realms.publickey;
+package org.sonatype.nexus.common.text;
 
-import java.security.PublicKey;
-
-public class MockPublicKey
-    implements PublicKey
+/**
+ * String helpers.
+ *
+ * @since 3.0
+ */
+public class Strings2
 {
-  private static final long serialVersionUID = -1748932588224311551L;
-
-  private String content;
-
-  public MockPublicKey(String content) {
-    this.content = content;
-  }
-
-  public String getAlgorithm() {
-    return "mock";
-  }
+  private Strings2() {}
 
-  public byte[] getEncoded() {
-    return content.getBytes();
+  public static boolean isEmpty(final String value) {
```

Comment:
FYI, this kind of method is typically named `isBlank` - any reason we don't use the one from Apache commons?